cmd/analyze.go

```diff
@@ -86,8 +89,11 @@ func (cmd *AnalyzeCmd) RunAnalyze(f factory.Factory, cobraCmd *cobra.Command, ar
 		return err
 	}
 
-	// Print warning
-	err = client.PrintWarning(generatedConfig, cmd.NoWarn, false, log)
+	// If the current kube context or namespace is different than old,
+	// show warnings and reset kube client if necessary
+	_, tty := terminal.SetupTTY(os.Stdin, os.Stdout)
```

Comment:
instead of initializing this everytime, please create a global variable in the kubectl package that you'll overwrite during tests

Reply:
done